PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/console/SparkScalaLivyConsoleRunConfiguration.kt

```diff
@@ -60,7 +62,7 @@ class SparkScalaLivyConsoleRunConfiguration(project: Project,
     override fun getState(executor: Executor, env: ExecutionEnvironment): RunProfileState? {
         val session = SparkSession(
                 name,
-                URI.create(JobUtils.getLivyBaseUri(cluster).toString()),
+                URI.create(if (cluster is LivyCluster) (cluster as LivyCluster).livyConnectionUrl else cluster.connectionUrl),
```

Comment:
I suppose the `SparkSession` second parameter type can be changed into `LivyCluster`